addons/website/static/src/js/menu/translate.js

```diff
@@ -350,4 +350,11 @@ registry.category("website_navbar_widgets").add("TranslatePageMenu", {
     Widget: TranslatePageMenu,
     selector: '.o_menu_systray:has([data-action="translate"])',
```

Comment:
In the meantime, try to adapt your commit message to match the [guideline](https://www.odoo.com/documentation/15.0/contributing/development/coding_guidelines.html#tag-and-module-name).

E.g. `[IMP] website: make translation components extensible` and some explanation after a blank line to describe why this is helpful.

Reply:
@bso-odoo okey, done